src/lib.rs

```diff
@@ -813,12 +813,37 @@ impl<T: Counter> Histogram<T> {
         self.record_n(value, T::one())
     }
 
+    /// Record `value` in the histogram, clamped to the range of the histogram.
+    ///
+    /// This method cannot fail, as any values that are too small or too large to be tracked will
+    /// automatically be clamed to be in range. Be aware that this *will* hide extreme outliers
+    /// from the resulting histogram without warning.
```

Comment:
We should also notify the user that this will *not* resize the histogram even if it is an auto-resize histogram -- we are clamping to the current bounds, not what they might be expanded to.